packages/protocol/migrations/27_governance.ts

```diff
@@ -49,61 +51,93 @@ module.exports = deploymentForCoreContract<GovernanceInstance>(
       const constitutionContractNames = Object.keys(constitution).filter(
         (contractName) => contractName !== 'proxy'
       )
+      let contract: any
```

Comment:
Can we have this strongly typed ?

Reply:
the let is not needed :)

not sure why the linter didn't complain.